packages/lesswrong/server/apolloServer.ts

```diff
@@ -241,7 +241,6 @@ export function startWebserver() {
     },
     //tracing: isDevelopment,
     tracing: false,
-    cacheControl: true,
```

Comment:
What's the effect of this change?

Reply:
From the commit message:

When we set up apollo-server, we were passing it `cacheControl: true`. This adds some metadata to the response in each graphql request, which lists each object-path saying that its max-age (for Apollo Engine, a caching middleware that we don't use and have never used) is zero.

Remove that flag. I believe this should have no effect on behavior except for making graphql requests slightly smaller.